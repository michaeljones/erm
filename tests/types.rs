extern crate erm;
extern crate im;
extern crate insta;
extern crate logos;
extern crate unindent;

mod common;

mod types {

    use common::eval;

    #[test]
    fn main_string_type_signature() {
        let src = r#"
        module Main exposing (..)

        main : List String -> String
        main args =
          "hello, world with type annotation"
        "#;
        let result = eval(src, None);
        insta::assert_snapshot!(result);
    }
}
