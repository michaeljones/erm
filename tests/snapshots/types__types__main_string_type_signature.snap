---
source: tests/types.rs
expression: result

---
hello, world with type annotation
